changelog.d/13470.misc

```diff
@@ -0,0 +1 @@
+Update matrix-synapse-ldap3 version in lockfile to v0.2.2.
```

Comment:
In the changelog for 1.62 I called this out as a bugfix, see https://github.com/matrix-org/synapse/blob/develop/CHANGES.md#synapse-1620rc3-2022-07-04. Can we do the same or adopt similar phrasing here?

Reply:
I've changed entry and re-purposed your language, let me know if this version is amenable.